app/views/coronavirus/reorder_sub_sections/index.html.erb

```diff
@@ -9,17 +9,17 @@
       href: coronavirus_page_path(slug: @page.slug)
     },
     {
-      text: "Reorder"
+      text: t("coronavirus.pages.sub_sections.reorder.title", section: @page.sections_title.downcase)
```

Comment:
Is it intentional that this doesn't match the previous text of "Reorder" ?

Reply:
Yeah, I followed the pattern of the other pages where the metadata text matches the page title.